### DISCUSSION TITLE
Steady Calculation in THM

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#23432] [Q&A General] [answered]
@joshuahansel I didn't add time term in the adding equation. And I prepare to write a steady input file to test the new `Component`. When I change the type of the `Excutioner` into `Steady` in `1D_flow_channel.i`. 
The terminal says:
```
*** ERROR ***
The following error occurred in the object "THM:problem", of type "THMProblem".

Vector tag 'TIME' does not exist.

This may occur if you have a TimeKernel in your problem but did not specify a transient executioner.
```
So does THM has the ability to perform steady-state calculation? If so, how can I run a steady test?

Ethan

**Comment by joshuahansel** [comment, marked as answer]
Hi Ethan,

This is because you have component(s) that add a time kernel (no matter what executioner you select). We find that actually doing a `Steady` calculation for flow equations is rarely successful due to nonlinear convergence difficulties; instead, we just run a transient to steady conditions. If you just have heat conduction, it should be fine though. If you really want to try, you can comment out the time kernels in the components that you use.

**Reply by Ethan-xj** [reply]
I see. Thank you:D
